R2API/Utils/Reflection.cs

```diff
@@ -923,5 +925,60 @@ public static class Reflection {
         }
 
         #endregion
+
+        public static System.Reflection.FieldInfo GetNestedField(Type type, string fieldName) {
```

Comment:
We could replace these two with a generic `GetNestedMember` and take in a generic type parameter that is constrained to a MemberInfo like here: https://github.com/risk-of-thunder/R2API/blob/1d9c3bdd53cbf50754c396c0845e2a0152ebdb45/R2API/Utils/Reflection.cs#L196

Reply:
That would be good. Would you like to persue that change or should I? I'm not familiar with creating generic type functions.